docs/guide/cache/fastly.md

```diff
@@ -0,0 +1,467 @@
+
+The example below extracts the generated vcl for version no. 11 of some service
+```
+curl -i  "https://api.fastly.com/service/[FASTLY_SERVICE_ID]/version/11/generated_vcl" -H "Fastly-Key: [FASTLY_API_TOKEN]" -H "Accept: application/json" > generated_vcl_11_raw
+cp generated_vcl_11_raw generated_vcl_11_json_only
+```
+
+Next, you need to edit generated_vcl_11_json_only in your favourite editor, remove anything before the json data and save.
+Then, do the same steps again for version no. 12 (or whatever version you want to diff version 11 against)
+
+Then replace \n in the files in order to give human-readable diffs:
+
+```
+cat generated_vcl_11_json_only |jq .content|perl -pe 's/\\n/\n/g' > generated_vcl_11_json_done
+cat generated_vcl_12_json_only |jq .content|perl -pe 's/\\n/\n/g' > generated_vcl_12_json_done
+```
+
+Finally, you can use GNU diff to and get a readable diff of the two versions:
+
+```
+diff -ruN generated_vcl_11_json_done generated_vcl_12_json_done
+```
+
+## Enabling basic-auth on Fastly.
+
+To enable basic-auth, use https://developer.fastly.com/solutions/examples/http-basic-auth as an example.
+
+Follow the steps below.
+
+Usernames and password can be stored inside vcl, but we'll instead store them in a [dictionary](https://docs.fastly.com/en/guides/working-with-dictionaries-using-the-web-interface#working-with-dictionaries-using-vcl-snippets)
+
+!!! note
+    In order to make this example work, you must run Ibexa DXP 3.3.16 or later, or 4.5.
+
+### Create the dictionary and activate the change:
+
+```
+fastly dictionary create --version=active --autoclone --name=basicauth
+fastly service-version activate --version=latest
+```
+
+Now, the Fastly configuration has a dictionary named `basicauth`. The benefit of using a dictionary than storing
+the usernames directly in vcl is that we can add or remove records without creating and activating new versions of the configuration.
+
+### Get the dictionary ID
+
+To add users to the dictionary, first get the dictionary ID.
+
+```
+fastly dictionary list --version=active
+
+Service ID: KlUh0J1fnw1JY1aEQ0up
+Version: 3
+ID: ltC6Rg4pqw4qaNKF5tEW
+Name: basicauth
+Write Only: false
+Created (UTC): 2023-07-03 10:33
+Last edited (UTC): 2023-07-03 10:33
+```
+
+In the example above, the ID is `ltC6Rg4pqw4qaNKF5tEW`.
+
+
+### Create a record in the dictionary ( user name and password )
+
+```
+fastly dictionary-item create --dictionary-id=ltC6Rg4pqw4qaNKF5tEW --key=user1 --value=foobar1
+```
+
+### List dictionary records
+
+You may see the records of a dictionary using:
+
+```
+fastly dictionary-item list --dictionary-id=ltC6Rg4pqw4qaNKF5tEW33
+```
+
+Now you have a dictionary storing a user name and password. The next thing to do is to alter the Fastly .vcl configration.
+and add the basic-auth support. 
+In this example, do that using snippets (https://docs.fastly.com/en/guides/about-vcl-snippets ) so that no changes
```

Comment:
```suggestion
This example, uses [snippets](https://docs.fastly.com/en/guides/about-vcl-snippets) so that no changes
```

Reply:
@MagdalenaZuba Comma should be removed to, right ?